sites/laserfocusworld/server/templates/index.marko

```diff
@@ -153,4 +154,4 @@ $ const adSlots = {
   <@footer>
     <endeavor-gam-ad-sticky-leaderboard name="lb2" refreshable=false aliases=aliases />
   </@footer>
-</theme-pennwell-website-section-layout>
+</theme-pennwell-website-section-layout>
```

Comment:
Ensure new lines are always added to the end of files. This can be configured in VSCode for all files you edit/save: see https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
commit a4c7725